src/components/volunteer-availabilities/VolunteerAvailability.jsx

```diff
@@ -1,13 +1,17 @@
 import React, { useState, useEffect } from 'react';
 import PropTypes from 'prop-types';
+import { Menu, Dropdown, Button, Space } from 'antd';
+import { DownOutlined } from '@ant-design/icons';
+import styles from './VolunteerAvailability.module.css';
 import HeatMap from './HeatMap';
+import { AFCBackend } from '../../util/utils';
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'AFCBackend' is defined but never used.